flang/docs/FlangDriver.md

```diff
@@ -15,17 +15,13 @@ local:
 ```
 
 There are two main drivers in Flang:
```

Comment:
It might be worth spending a paragraph or section explaining the historical role of `flang-new`, if only so that people understand why it's (eventually) not available anymore.

Reply:
Good point 👍🏻